pkg/querier/queryrange/shard_resolver.go

```diff
@@ -0,0 +1,153 @@
+package queryrange
+
+import (
+	"context"
+	"fmt"
+	math "math"
```

Comment:
nit: did you mean to include the name prefix here?

Reply:
Thanks (I did not)